02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,39 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+  <title>Khadijah's Bio</title>
 </head>
 <body>
-  <h1>Indian Restaurant</h1>
+  <h1><u>Khadijah's Bio</u></h1>
+  <a ref="">I own a company called Best Lead Generation that
+    specializes in providing <div> business-to-business lead generation
+    services to companies in order to help <div>grow their business.</a>
+
+    <h3><u>Things I like:</u><h3>
+    <ul>
+    <li> Reading
+    <li> Hiking
+    <li> Baking
+    <li> Cooking
+    <li> Coding
+    </ul>
+
+<h3><u>Things I don't like:</u><h3>
```

Comment:
Don't forget to tab over on the `<h3>` tags. This is for proper code formatting and human readability. Remember, the browser doesn't care how you format your HTML or CSS code, but it is easier to read if it is formatted with indents.